src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form"
+      data-qa="big"
+    >
```

Comment:
The `<form>` tag is self-closing, which is incorrect in HTML. Forms should have an opening and closing tag. Consider changing `<form />` to `<form>` and ensure it has a corresponding `</form>` closing tag.